database.php

```diff
@@ -0,0 +1,7 @@
+<?php
+$connection = mysqli_connect("localhost", "root", "", "books_db");
```

Comment:
**Observations**: The code does not check if the database connection is successful before using it.

**Recommendations**: Ensure that the database connection is successful before proceeding with any queries. This can prevent unexpected errors during database operations.

```php
if (!$connection) {
    die("Failed to connect to the database: " . mysqli_connect_error());
}
```

Or in the object-oriented style:

```php
if ($connection->connect_error) {
    die("Connection failed: " . $connection->connect_error);
}
```